docs/access.rst

```diff
@@ -117,3 +129,6 @@ To change language in which **Kolibri** user interface is displayed, follow thes
 #. Select the **Change language** option.
 #. Choose the desired language.
 #. Click **Confirm** and Kolibri will be displayed in the selected language!
+
+.. warning::
+  The default language configured for content in your browser preferences might override the language you choose in this step. To ensure that Kolibri displays in the desired language, make sure to configure it as default in the browsers of all the devices that will be used to view Kolibri content.
```

Comment:
Is this really true? Is there an issue open about it? Selecting language from HTTP request headers sent by the browser isn't the only way to do things :)

(I'm sending some changes to this paragraph btw)

Reply:
https://github.com/learningequality/clearinghouse/issues/219, and seen in practice while performing the upgrade with VF partners (cc @jamalex).

Reply:
Thanks, posted a comment there.